api/config/config_local.json

```diff
@@ -163,6 +163,7 @@
     "usersPerAuditTask": 20,
     "usersPerSynchronizeAccessTask": 50,
     "usersPerCheckInitialCreditsExpirationTask": 50,
+    "usersPerAccessExpirationEmailTask": 500,
```

Comment:
trying a larger number for this one because I expect each check will be fast.  We'll see!